src/SourceBuild/content/repo-projects/source-build-reference-packages.proj

```diff
@@ -8,16 +8,24 @@
     <!-- SBRP builds before Arcade so it also needs the bootstrap Arcade version -->
     <UseBootstrapArcade>true</UseBootstrapArcade>
 
+    <!-- SBRP builds the reference SDKs so it needs the bootstrap SDK versions -->
+    <UseBootstrapSdks>true</UseBootstrapSdks>
```

Comment:
I would make this property name more specific and maybe even introduce one property per msbuild sdk, i.e. `UseBootstrapNoTargets` and `UseBootstrapTraversal`. We will eventually add more msbuild sdks that needs this distinction, i.e. the `Microsoft.NET.Sdk` and the `Microsoft.NET.WindowsDesktop.Sdk`. Overall the plan is to get rid of all these and just let the nuget sdk resolver do this automatically.

Reply:
I've actually simplified this as I realized that the overrides will disappear when the package hasn't been built by a dependency (giving the correct behavior to SBRP by default).